### PR TITLE
[Finishes #163934910] application setup tests

### PR DESCRIPTION
**Pull request by maxwelbarno**
#### What does this PR do?
Tests if the application scales to different execution environments: testing and development
#### Description of Task to be completed?
Passing tests
#### How should this be manually tested?
On your terminal:
```git clone https://github.com/kipruto/politico-api.git```
```cd politico-api```
Run ```virtualenv venv``` to set up environment
Run ```pip install -r requirements.txt```
```pytest --cov=app tests/v2```
#### What are the relevant pivotal tracker stories?
#163934910

### CODE REVIEW

File: app/api/v2/db.py [hunk 1]
```diff
@@ -84,31 +84,30 @@ def create_queries():
...
+                    FOREIGN KEY (userId) REFERENCES users (id)
+                    ON DELETE CASCADE
+                    ON UPDATE CASCADE );"""
 
     votes = """CREATE TABLE IF NOT EXISTS votes(
                     id SERIAL PRIMARY KEY NOT NULL, 
-                    createdOn TIMESTAMP NULL DEFAULT NOW()
-                    createdBy VARCHAR(50) NOT NULL,
+                    createdOn TIMESTAMP NULL DEFAULT NOW(),
+                    createdBy INTEGER NOT NULL,
```
Comment (petermwash) [inline app/api/v2/db.py:94]: Kindly explain why `createdBy` should be an INTEGER.

Comment (maxwelbarno) [reply app/api/v2/db.py:94]: 'CreatedBy' is an integer because it references the user id. It is a Foreign Key

**Review by petermwash — APPROVED** [review]
LGTM.

**Comment by maxwelbarno** [comment]
Thanks. Closed

**Pull request merged**